src/Tools/NodeDocumentationMarkdownGenerator/Program.cs

```diff
@@ -11,16 +11,28 @@ namespace NodeDocumentationMarkdownGenerator
 {
     class Program
     {
-        internal static IEnumerable<FileInfo> DynamoDirectoryAssemblyPaths;
-        static void Main(string[] args)
+        private static IEnumerable<FileInfo> dynamoDirectoryAssemblyPaths;
```

Comment:
curious about this change? Are you trying to make this faster?

Reply:
it was more so that we could unit test without `Program.Main`, the AssemblyHandler uses the dynamoDirectoryAssemblyPaths so if main hadn't been executed the assembly handler would fail here:
https://github.com/DynamoDS/Dynamo/pull/11725/files#diff-4ec66792a154bbfd3bc76223fc2de6df6a91cc71eebc43d29d427b0a96ada7a2R294